frontend/src/components/widgets/wordCard/WordCard.jsx

```diff
@@ -71,12 +71,12 @@ const WordCard = ({
 }
 
 export default WordCard;
-
 WordCard.propTypes={
 //necesarry fields
   status: PropTypes.string.isRequired,
```

Comment:
Word card status can only have three values (not any string) 
optionalEnum: PropTypes.oneOf(['News', 'Photos']) change it to something like this.